.github/workflows/build.yml

```diff
@@ -67,3 +67,18 @@ jobs:
         ALIYUN_DOCKER_USERNAME: ${{ secrets.ALIYUN_DOCKER_USERNAME }}
         ALIYUN_DOCKER_PASSWORD: ${{ secrets.ALIYUN_DOCKER_PASSWORD }}
       run: bash build-image.sh
+  build-mongoDB:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        service: ["mongoDB"]
```

Comment:
should be "mongo", not "mongoDB". see [docker-compose.yml](https://github.com/laradock/laradock/blob/master/docker-compose.yml#L667)